packages/clerk-js/src/core/resources/User.ts

```diff
@@ -209,6 +209,10 @@ export class User extends BaseResource implements UserResource {
 
   setProfileImage = (params: SetProfileImageParams): Promise<ImageResource> => {
     const { file } = params || {};
+
+    if (!file) {
```

Comment:
:question: Is this check enough? Should we explicitly check that the `file` parameter is set to `null` before we trigger a DELETE request?

Reply:
@gkats what edge case do you have in mind?

Reply:
I don't have an edge case in mind. It's just an API choice.

```js
// No file in the parameters, is this intentional?
user.setProfileImage();

// vs

// File is passed and set to null
user.setProfileImage({ file: null });
```

I'm not sure how TS will behave with the above, but in plain JS, undefined and null will be different.

cc @SokratisVidros

Reply:
Typescript should complain but not everyone will be using typescript, so maybe it should indeed be explicit.

Reply:
We need to be explicit about this, otherwise any improper call invocation would delete the image. Lets check for `=== null`

Reply:
☝️